packages/create-mc-app/package.json

```
@@ -25,6 +25,6 @@
     "semver": "7.3.5"
   },
   "engines": {
-    "node": ">=14"
+    "node": "^14 || ^16 || ^17"
```

Thanks. Nit 
```suggestion
    "node": "14.x || 16.x || 17.x"
```
As that's the notation we often use in the repo. Mainly to keep it simple. Anybody understands a `14.x` while a  `^` usually leads to people going "Hot dang what was `~` and `^` again?" :)